style.css

```diff
@@ -0,0 +1,228 @@
+body {
+    height: 100vh;
+    width: 100vw;
+    background-color: darkred;
+    margin: 0;
+}
+
+.header1 {
+    background-color: black;
+    height: 20vh;
+    width: 100vw;
+    /* position: absolute; */
+    /* top: 0; */
+
+}
+
+.headerImage {
+    position: relative;
+    left: 50%;
+    transform: translateX(-50%);
+    height: 84%;
+    width: auto;
+}
+
+main {
+    height: 80vh;
+    font-size: 0;
+}
+
+.playerBoxBorder{
+    height: 100%;
+    width: 25%;
+    display: inline-block;
+}
+
+.playerBox {
+    position: relative;
+    top: 7%;
+    left: 50%;
+    transform: translateX(-50%);
+}
+
+.playerBox1 {
+    width: 15vw;
+    background-color: darkslategray;
+    border-radius: 8px;
+    border: 1px solid red;
+    /* position: relative; */
+    height: 60vh;
+    text-align: center;
+    /* margin-left: 25px; */
+    /* top: 5vh; */
+    display: inline-block;
+    vertical-align: top;
+}
+
+.statsBox {
+    position: relative;
+    top: 13%;
+    height: 60%;
+    color: #331c14;
+    /* background-color: lime; */
+}
+
+#player1 {
+    width: 6em;
+    height: 2em;
+}
+
+.forfeitButton {
+    position: relative;
+    top: 13%;
+}
+
+.playerBox2 {
+    width: 15vw;
+    background-color: darkslategray;
+    border-radius: 8px;
+    border: 1px solid red;
+    /* position: relative; */
+    height: 60vh;
+    text-align: center;
+    /* left: 82vw; */
+    /* bottom: 60vw; */
+    display: inline-block;
+    vertical-align: top;
+
+}
+.pieceDisplay1 {
+
+    background-color: darkslategray;
+    /* height: 20vh; */
+    width: 10vw;
+    position: relative;
+    top: 4vh;
```

Comment:
watch all this usage of vw/vh/vmin, it will make it very hard to resize things later on.

![close-up-portrait-head-shiba-inu-dog-black-background-looks-curious-camera-front-view-sad-eyes-74358604](https://user-images.githubusercontent.com/10343746/40747217-728eaef0-6411-11e8-8ba4-8af46a2a77f4.jpg)